boot/freeldr/freeldr/include/arch/pc/hardware.h

```diff
@@ -23,6 +23,22 @@
 #define TAG_HW_RESOURCE_LIST    'lRwH'
 #define TAG_HW_DISK_CONTEXT     'cDwH'
 
+/*
+ * Unfortunately, the ioaccess.h header doesn't define the string I/O operations
+ */
+#define READ_PORT_BUFFER_UCHAR(port, buffer, count) \
```

Comment:
Regarding the "stop them being pulled from...." : Since freeldr uses the minihal library anyway, what would be the reason to stop using them from there?

Reply:
I suggest to exclude the portio.c file from the mihinal library since it looks like ioaccess.h provides enough functionality to make use of I/O ports.